cmd/go-tpc/tpch.go

```diff
@@ -67,6 +87,23 @@ func executeTpch(action string) {
 	w.OutputStats(true)
 }
 
+func getServerVersion(db *sql.DB) (string, error) {
+	var version string
+	err := db.QueryRow("SELECT VERSION()").Scan(&version)
+	return version, err
+}
+
+func setQueryTuningVars(db *sql.DB, ver util.SemVersion) error {
+	for _, v := range queryTuningVars {
+		if isSysVarSupported(ver, v.name) {
```

Comment:
Shall we add a corresponding `else` branch that prints a warning or info log of "Unsupported query tuning var %s for version %s"?

Reply:
LGTM, I'll add the log.